packages/Avalonia/AvaloniaBuildTasks.targets

```diff
@@ -68,14 +76,17 @@
           Condition="('@(AvaloniaResource->Count())' &gt; 0) or ('@(AvaloniaXaml->Count())' &gt; 0)"
           >
     <ItemGroup>
-        <AvaloniaResource Include="@(AvaloniaXaml)"/>
+      <AvaloniaResource Include="@(AvaloniaXaml)" />
     </ItemGroup>
     <GenerateAvaloniaResourcesTask
       Condition="'$(_AvaloniaUseExternalMSBuild)' != 'true'"
       Output="$(AvaloniaResourcesTemporaryFilePath)"
       Root="$(MSBuildProjectDirectory)"
       Resources="@(AvaloniaResource)"
       ReportImportance="$(AvaloniaXamlReportImportance)"/>
+    <ItemGroup Condition="'$(_AvaloniaUseExternalMSBuild)' != 'true'">
```

Comment:
I think those conditions shouldn't be there. When we are triggering a separate msbuild process (so it doesn't lock the in-solution tasks dll file) it would still write to the same file

Reply:
The condition should pass for that external msbuild process, which will do the write. Keeping it will likely avoid a double write (not that it really matters).

I considered splitting each target into two (internal and external) to avoid repeating the conditions, but it seemed a bit overkill :)